src/styles/index.scss

```diff
@@ -4,0 +1,7 @@
+@import '/src/styles/blocks/variables';
+@import '/src/styles/blocks/page';
+@import '/src/styles/blocks/card';
```

Comment:
Ensure that the import path for the card styles is correct and relative to the SCSS file location. The current path '/src/styles/blocks/card' should be './blocks/card' to align with the project structure.